src/App.vue

```diff
@@ -115,18 +122,34 @@ export default {
     let types_by_host = {};
     _.each(buckets, (v) => {
         types_by_host[v.hostname] = types_by_host[v.hostname] || {};
-        if(v.type == "afkstatus") {
-            types_by_host[v.hostname].afk = true;
-        } else if(v.type == "currentwindow") {
-            types_by_host[v.hostname].window = true;
-        }
+        // The '&& true;' is just to typecoerce into booleans
+        types_by_host[v.hostname].afk |= v.type == "afkstatus";
+        types_by_host[v.hostname].window |= v.type == "currentwindow";
+        types_by_host[v.hostname].android |= v.type == "currentwindow" && this.isAndroidApp;  // Use other bucket type ID in the future
```

Comment:
My idea is to list the views that are available given whatever data the user is collecting, this would mean a seperate editor-view, as requested in the linked issue.